src/main/java/tutoraid/model/lesson/Students.java

```diff
@@ -1,40 +1,43 @@
 package tutoraid.model.lesson;
 
-import static java.util.Objects.requireNonNull;
-import static tutoraid.commons.util.AppUtil.checkArgument;
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+import tutoraid.model.student.Student;
 
 /**
- * Represents a Lesson's timing in TutorAid.
+ * Represents a Lesson's students in TutorAid.
  * Guarantees: immutable; is valid as declared in {@link #isValidStudents(String)}
```

Comment:
Since the method `isValidStudents` does not exist anymore in this file, it may be better to remove this guarantee in this JavaDoc header comment, or to re-implement the `isValidStudents` method?

Reply:
Right ok will remove it.